functions/graphql/schema/flightRecord.js

```diff
@@ -63,7 +63,7 @@ module.exports.schema = gql`
       date: Date
       flightTypeId: ID
       simulators: [FlightSimulatorInput]
-    ): FlightRecord @auth(requires: [director])
+    ): Boolean @auth(requires: [director])
```

Comment:
Why does this need to be a Boolean?

Reply:
Just to show if it was updated successfully or not, but I'm not attached to it being boolean if you want a more useful result.

Reply:
Having it be a Flight Record that is returned (and throwing an error if it fails) makes it easier for me to update the UI with the results of the mutation.
```suggestion
    ): FlightRecord @auth(requires: [director])
```